linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_kmod/oval/ubuntu2004.xml

```diff
@@ -0,0 +1,40 @@
+<def-group>
```

Comment:
you can drop this, we are using the templated oval, unless you see something wrong with it

Reply:
The templated oval does not check the correct flags. See below for what the templated results are producing vs DISA Benchmark